chaco/__init__.py

```diff
@@ -8,4 +8,6 @@
 except ImportError:
     __version__ = "not-built"
 
-__requires__ = ["traits", "traitsui", "pyface", "numpy", "enable"]
+__requires__ = [
+    "traits>=6.2.0", "traitsui", "pyface>=7.2.0", "numpy", "enable>=5.2.0"
```

Comment:
I should check that a release candidate actually counts for >= .  Otherwise even with the enable 5.2.0rc2 out, there will not be available packages on PyPI satisfying the requirements when we make the rc for chaco.

If that is the case, I will unpin this requirement until after the chaco rc is made, and after enable 5.2.0 is fully released, but before chaco 5.0.0 is release